docs/features/global_tools.md

```diff
@@ -0,0 +1,177 @@
+```
+Results in the following entry in the manifest:
+```toml
+[envs.snakemake]
+channels = ["conda-forge", "bioconda"]
+dependencies = { snakemake = "*" }
+exposed = { snakemake = "snakemake" }
+```
+
+More information on channels can be found [here](../advanced/channel_priority.md).
+
+### Exposed
+The exposed binaries are the ones that will be available in the system `PATH`.
+This is useful when the package has multiple binaries, but you want to get a select few, or you want to expose it with a different name.
+For example, the `python` package has multiple binaries, but you only want to expose the interpreter.
+Running:
+```
+pixi global expose add --environment python py3=python3
+```
+will create the following entry in the manifest:
+```toml
+[envs.python]
+channels = ["conda-forge"]
+dependencies = { python = ">=3.10,<3.11" }
+exposed = { py3 = "python3" }
+```
+There is some added automatic behavior, if you install a package with the same name as the environment, it will be exposed with the same name.
+Even if the binary name is only exposed through dependencies of the package
+For example, running:
+```
+pixi global install ansible
+```
+will create the following entry in the manifest:
+```toml
+
+[envs.ansible]
+channels = ["conda-forge"]
+dependencies = { ansible = "*" }
+exposed = { ansible = "ansible" } # (1)!
+```
+
+1. The `ansible` binary is exposed even though it is installed by a dependency of `ansible`, the `ansible-core` package.
+
+### Dependencies
+Dependencies are the **Conda** packages that will be installed into your environment. For example, running:
+```
+pixi global install "python<3.12"
+```
+creates the following entry in the manifest:
+```toml
+[envs.vim]
+channels = ["conda-forge"]
+dependencies = { python = "<3.12" }
+# ...
+```
+Typically, you'd specify just the tool you're installing, but you can add more packages if needed.
+Defining the environment to install into will allow you to add multiple dependencies at once.
+For example, running:
+```shell
+pixi global install --environment my-env git vim python
+```
+will create the following entry in the manifest:
+```toml
+[envs.my-env]
+channels = ["conda-forge"]
+dependencies = { git = "*", vim = "*", python = "*" }
+# ...
+```
+
+You can `add` a dependency to an existing environment by running:
+```shell
+pixi global install --environment my-env package-a package-b
+```
+This will be added as dependencies to the `my-env` environment but won't auto expose the binaries from the new packages.
+
+You can `remove` dependencies by running:
+```shell
+pixi global remove --environment my-env package-a package-b
+```
+
+
+### Example: Creating a Data Science Sandbox Environment
+You can create an environment with multiple tools using the following command:
+```shell
+pixi global install --environment data-science --expose jupyter=jupyter --expose ipython=ipython jupyter numpy pandas matplotlib ipython
```

Comment:
Use fewer packages to avoid that the reader has to scroll

Reply:
Done